libcaf_io/test/basp_udp.cpp

```diff
@@ -829,6 +828,7 @@ CAF_TEST(actor_serialize_and_deserialize_udp) {
           std::vector<actor_id>{}, msg);
 }
 
+/*
```

Comment:
Remove if it's obsolete now. Otherwise, please provide comments when this block gets "re-activated".